Directory.Build.Props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <Product>GitHub Extension for Visual Studio</Product>
-    <Version>2.5.5.0</Version>
+    <Version>2.5.6.0</Version>
```

Comment:
Do you need to manually version? There are better ways to do this :)

Reply:
Oh really? Please do elucidate! What would be a better way to do this?

Reply:
I've always hated version stamping. I'd be interested in finding the least-hacky way! 😉

Reply:
I have a project that matches this one very closely -- xUnit for Devices. It has a VSIX for its templates and auto-versions.

https://github.com/xunit/devices.xunit

Here are some key things:

1. Uses Nerdbank.GitVersioning to get a version for each commit. That handles the csproj's.
2. Build scripts update other xml assets to match: https://github.com/xunit/devices.xunit/blob/master/.vsts-shared.yml#L31-L36